index.html

```diff
@@ -88,7 +88,7 @@ <h3><em>Convert a Scratch project to HTML</em></h3>
         id="other-htmlify-btn"
         form="options"
         type="submit"
-      >
+      title="HTMLify">
```

Comment:
Links with text do not need titles with redundant information, even for accessibility